tests/console/verify_and_prepare_install.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check if iso was cloned and switch to X
+# Maintainer: Joachim Rauch <jrauch@suse.com>
+# Tags: boo#1040749
+
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+
+sub run {
+    assert_script_run 'ls /dev | grep vd*';
+    assert_script_run 'ls /mnt/';
+    type_string "logout\n";
+    select_console 'x11';
```

Comment:
The purpose of this file is to provide a clean transition from the console test to x11 since the base btrfs test leaves a logged in console after finishing which lead to problems when wanting to shut down via x11.